docs/content/en/docs/install/install.md

```diff
@@ -80,14 +74,14 @@ as well as for upgrades.
 Some helpful hints:
 
 * Use the `--version <version>` flag on the
-  `helm upgrade --install` command line to specify a different Keptn version.
+  `helm upgrade --install` command line to specify a Keptn version
+  other than the latest.
```

Comment:
let's add an example of a `helm upgrade --install` with the `--version` flag being used

Reply:
done